### PR TITLE
Add djLint to lint HTML

### PR DESCRIPTION
**Pull request by johannaengland**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -41,3 +41,8 @@ output-format = "full"
 
 [tool.ruff.lint]
 select = ["E9", "F63", "F7", "F82"]
+ignore = ["E731"]
```
Comment (lunkwill42) [inline pyproject.toml:44]: What does this have to do with djLint?

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -14,3 +14,20 @@ jobs:
...
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+
+    - name: Install dependencies
+      run: |
+        pip install -U pip
+        pip install djlint
```
Comment (lunkwill42) [inline .github/workflows/lint.yml:30]: You will probably encounter the same issue with using pip to install tools globally once this workflow runs on an Ubuntu 24.04 worker in GitHub actions.  Perhaps consider the pipx route also here?

**Review by lunkwill42 — CHANGES_REQUESTED** [review]
I was never part of the djLint discussions, but would like to take part in a discussion of what linter rules we would want to enable for this tool.

The PR looks mostly fine to me, just a couple of potential quirks...

File: pyproject.toml (hunk 1 above)
Comment (johannaengland) [reply pyproject.toml:44]: It doesn't. This is a leftover from when it was based on a different version of main and I incorrectly rebased it, will fix this now

File: .github/workflows/lint.yml (hunk 2 above)
Comment (johannaengland) [reply .github/workflows/lint.yml:30]: I won't since one of the solutions to the global pip install is using the `Set up Python` step before (see: https://github.com/actions/runner-images/issues/10781#issuecomment-2414587300)
